components/apply/PositionPreference.tsx

```diff
@@ -0,0 +1,65 @@
+import { FC, useState, useEffect } from "react";
+import SelectInput from "@components/common/SelectInput";
+import MEMBER_ROLES from "@constants/member-roles.json";
```

Comment:
Can we declare MEMBER_ROLES as a const in this file instead of using a json?

This way, we can make a `MemberRole` type for it and use the type in the state declarations.

Reply:
i.e.

```
type MemberRoles = {
    readonly label: string;
    readonly value: string;
}

const MEMBER_ROLES: MemberRoles[] = [
    { label: "Project Developer", value: "1" },
    { label: "Product Designer", value: "2" },
    ...
];
```